products/rhel10/profiles/ism_o_secret.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
```

Comment:
Profile files need to have the `.profile` extension. If they have `.yml` extension they are silently ignored by the build system and don't get to the built data stream.